make-distribution.sh

```diff
@@ -159,7 +159,7 @@ fi
 # Build uber fat JAR
 cd "$SPARK_HOME"
 
-export MAVEN_OPTS="-Xmx2g -XX:MaxPermSize=512M -XX:ReservedCodeCacheSize=512m"
+export MAVEN_OPTS="${MAVEN_OPTS:--Xmx2g -XX:MaxPermSize=512M -XX:ReservedCodeCacheSize=512m}"
```

Comment:
For posterity (since this initially confused me): This is the Bash syntax for parameter substitution with a default value.

Reference: http://www.tldp.org/LDP/abs/html/parameter-substitution.html